app/2.0/docs/devguide/dom-template.md

```diff
@@ -76,19 +91,148 @@ class MyElement extends Polymer.Element {
        <div>I've got a string template!</div>`
   }
 }
+
 customElements.define('my-element', MyElement);
 ```
 
-When using a string template, the element doesn't need to provide an `is` getter (however, the tag
-name still needs to  be  passed as the first argument to `customElements.define`).
+[See a working example in Plunker](http://plnkr.co/edit/AzkMDJ?p=preview).
+
+**When using a static `template` getter, the element doesn't need to provide an `is` getter.**
+However, the tag name still needs to be passed as the first argument to
+`customElements.define`, as in the example above.
+{.alert .alert-info}
+
+Polymer 2.4+ provides the `Polymer.html` function that can be applied to a template literal to
+produce an `HTMLTemplateElement`. 
 
-### Inherited templates {#inherited-templates}
+Use the Polymer.html function to convert a template literal to an HTMLTemplateElement: { .caption }
+
+```js
+class MyElement extends Polymer.Element {
+
+  static get template() {
+    return Polymer.html`<style>:host { color: blue; }</style>
+       <h2>String template</h2>
+       <div>I've got a string template!</div>`;
+  }
+}
+customElements.define('my-element', MyElement);
+```
+
+[See a working example in Plunker](http://plnkr.co/edit/fFTGJf?p=preview).
+
+Some text editors support HTML code highlighting in template literals tagged with a function
+called `html`. To enable HTML code highlighting in such text editors, declare a constant called
+`html` to hold the `Polymer.html` function. Use the `html` constant in the template getter.
+
+Declare a constant called html to hold the Polymer.html function: { .caption }
+
+```js
+const html = Polymer.html;
+
+class MyElement extends Polymer.Element {
+
+  static get template() {
+    return html`<style>:host { color: blue; }</style>
+       <h2>String template</h2>
+       <div>I've got a string template!</div>`;
+  }
+}
+customElements.define('my-element', MyElement);
+```
+
+### Inherited templates {#inherit}
 
 An element that extends another Polymer element can inherit its template. If the element doesn't
-provide its own DOM template (using either a `<dom-module>` or a string template), Polymer uses the
-same template as the superclass, if any.
+provide its own DOM template (using either a `<dom-module>` or a static template object), Polymer
+uses the same template as the superclass, if any.
```

Comment:
I think this list of items needs some introduction, like: 

The `Polymer.html` helper introduced in Polymer 2.4 makes template extension a little simpler, so this section shows examples with `Polymer.html` (usable on 2.4 and later) and without (usable on all 2.x versions).

Or... something.

Reply:
- [ ] intro to list